portal/views/views_instances.py

```diff
@@ -89,6 +89,25 @@ def view_instance(name):
                                 instance_log=instance_log)
 
 
+@app.route('/instances-log-xhr/<name>/<lines>', methods=['GET'])
```

Comment:
here I need to pass in the name and lines variables from VueJS. At some point I was trying to use 'POST' method to shorten the URL, but I kept having 400 problem with POST.

basically this is where I use the API to get the new log. let me know if the function and variable names should be changed